includes/environment.inc

```diff
@@ -246,7 +236,7 @@ function drush_is_local_host($host) {
  * Determine whether current OS is a Windows variant.
  */
 function drush_is_windows($os = NULL) {
-  return strtoupper(substr($os ?: PHP_OS, 0, 3)) === 'WIN';
+  return strtoupper(substr(_drush_get_os($os), 0, 3)) === 'WIN';
```

Comment:
If we stop using `LOCAL` in backend.inc, then this can stay as it used to be

Reply:
I was concerned that would be a big deal, but it looks like `_drush_backend_generate_command()` is the only function using LOCAL anymore.  My only lingering thought remains in keeping the `_drush_get_os()` abstraction to remain consistent with the other OS checks.

I'll go back and remove LOCAL and update `_drush_get_os()` one more time.